lib/migration.c

```diff
@@ -442,14 +442,14 @@ migration_region_access(vfu_ctx_t *vfu_ctx, char *buf, size_t count,
             vfu_log(vfu_ctx, LOG_WARNING,
                     "bad access to dead space %#lx-%#lx in migration region",
                     pos, pos + count - 1);
-            return -EINVAL;
+            return ERROR_INT(EINVAL);
         }
 
         pos -= migr->data_offset;
         if (is_write) {
             ret = migr->callbacks.write_data(vfu_ctx, buf, count, pos);
```

Comment:
Check that errno is non-zero?

Reply:
as per the other set, I don't think it really makes sense to try and check this.